eagledns/src/main/java/se/unlogic/standardutils/crypto/EncryptionUtils.java

```diff
@@ -96,18 +97,12 @@ public static String hash(File file, String algorithm, ProgressMeter progressMet
 		}
 	}
 
-	public static String mysqlPasswordHash(String string){
+	public static String mysqlPasswordHash(String string) throws UnsupportedEncodingException{

```

Comment:
@CathrineTeng , This should be still try catch UnsupportedEncodingException and wraps it in a RuntimeException. Is there any reason change to throws UnsupportedEncodingException ?